File: Semester3/Advanced Programming Methods/Labs/lab3/src/domain/validators/ValidatorException.java
package domain.validators;

public class ValidatorException extends Exception {

    /**
     * Constructor for StudentException class
     * @param err-string for output errors
     */
    public ValidatorException(String err){
        super(err);
    }

    /**
     * Default constructor
     */
    public ValidatorException() {}
}

File: Semester4/SGBD/Tema_lab1/Lab1_CRUD_2Tables/Lab1_CRUD_2Tables/Domain/Validators/ValidatorException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab1_CRUD_2Tables.Domain.Validators
{
    class ValidatorException : Exception
    {
        /// <summary>
        /// Constructor for Validator exception class
        /// </summary>
        /// <param name="err">error to be printed</param>
        public ValidatorException(string err):base(err){

        }
    }
}
